osu.Game/Skinning/DefaultFont.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Skinning.Components;
+
+namespace osu.Game.Skinning
+{
+    /// <summary>
+    /// The type of built-in font to use for <see cref="DefaultTextSkinComponent"/>.
+    /// </summary>
+    public enum DefaultFont
+    {
+        Venera,
+        Torus,
+        TorusAlt,
+        Inter
+    }
```

Comment:
- Why does this enum even exist if `Typeface` exists? Would get rid of the necessity to map the enums across and whatever.
- (The following applies to whichever enum survives) Can these have `[Description]`s or something? The dropdown at present looks really quite ugly what with the enum names printed directly to the user. Almost looks like a debug option or something.
  
  ![image](https://user-images.githubusercontent.com/20418176/205460327-c07a68ec-d18b-4173-828d-8b1de73e7ac7.png)

Reply:
Somehow missed the fact that it already existed. The only issue is that venera requires non-default font weight, but it still looks cleaner than dublicated enum.